robot_smach_states/src/robot_smach_states/util/startup.py

```diff
@@ -26,24 +27,25 @@
 
 
 def startup(statemachine_creator, statemachine_args = (), initial_state=None, robot_name='', challenge_name=None, argv=sys.argv):
-    '''
+    """
     :param statemachine_creator: a function that outputs a statemachine.
         The function should take a robot as its first input.
     :param statemachine_args: A list of arguments. If the statemachine_creator
         function takes any arguments besides robot these can be placed here.
     :param initial_state the state to start the state machine in.
         Can be supplied as command line argument
-    :param robot_name name of the robot to pass to the state machine'''
-
+    :param robot_name name of the robot to pass to the state machine
+    """
+    t_start = time.time()
```

Comment:
why import `time` package and use this instead of `rospy.get_time()` (returns float in seconds)?